consensus/src/epoch_manager.rs

```diff
@@ -1451,15 +1449,95 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
                 .ok_or_else(|| anyhow::anyhow!("Epoch state is not available"))?;
             let proof_cache = self.proof_cache.clone();
             let quorum_store_enabled = self.quorum_store_enabled;
-            let quorum_store_msg_tx = self.quorum_store_msg_tx.clone();
+            let quorum_store_verified_msg_tx = self.quorum_store_verified_msg_tx.clone();
+            let quorum_store_unverified_msg_tx = self.quorum_store_unverified_msg_tx.clone();
             let buffered_proposal_tx = self.buffered_proposal_tx.clone();
-            let round_manager_tx = self.round_manager_tx.clone();
+            let round_manager_verified_tx = self.round_manager_verified_tx.clone();
+            let round_manager_unverified_tx = self.round_manager_unverified_tx.clone();
             let my_peer_id = self.author;
             let max_num_batches = self.config.quorum_store.receiver_max_num_batches;
             let max_batch_expiry_gap_usecs =
                 self.config.quorum_store.batch_expiry_gap_when_init_usecs;
             let payload_manager = self.payload_manager.clone();
             let pending_blocks = self.pending_blocks.clone();
+
+            if self.config.optimistic_sig_verification_for_order_votes
```

Comment:
what's the benefit of a different channel for unverified messages? I think ideally we have a new type for unverified signature instead of a different channel or boolean to indicate whether it's verified or not

Reply:
> what's the benefit of a different channel for unverified messages? 
The current channel has the message type as VerifiedEvent. I didn't like sending an unverified vote as a VerifiedEvent along with a boolean stating that the message as unverified. So, had to open another channel for UniverifiedEvents.

Reply:
I think type safety is good enough, one of the enum in the VerifiedEvent contains unverified sync info too. if we have  a UnverifiedSignature or something we should be good. (plus we can rename the VerifiedEvent to something more mixed)

Reply:
If you prefer having a single channel, then I can rename `VerifiedEvent` to contain (msg, bool) instead of just containing msg. The bool flag indicates whether the msg is verified.